Frontend/ui-customer/index.js

```diff
@@ -252,6 +252,9 @@ function refreshFlow(){
       (token != "__failed") && 
       (token != "(null)") &&
       ((currentState == "RideStarted") || currentState == "RideAccepted")){
+    if(window.storeCallBackMessageUpdated){
+      window.__PROXY_FN[window.storeCallBackMessageUpdated] = undefined;
+    }
```

Comment:
don't use PROXY_FN use our defined function to make it more clear and so that its easier to debug and for others to understand